.env

```diff
@@ -1,26 +1,33 @@
-DYDX_URL=https://api.dydx.exchange
-
+# Gas Management
 GAS_STATION_URL=http://ethgasstation.info/json/ethgasAPI.json
 GAS_PRICE_MULTIPLIER=2
 GAS_PRICE_UPDATE_FREQUENCY_SEC=120
-INITIAL_GAS_PRICE=30000000000
+GAS_INITIAL_PRICE=30000000000
 GAS_REQUEST_TIMEOUT_MS=10000
 GAS_PRICE_ADDITION=1
 
-LIQUIDATION_KEY_EXPIRATION_SEC=300
-LIQUIDATION_COLLATERAL_PREFERENCES=0,3,1,2
-LIQUIDATION_OWED_PREFERENCES=0,3,1,2
-MIN_LIQUIDATOR_ACCOUNT_COLLATERALIZATION=0.5
-MIN_VALUE_LIQUIDATED=100e36
-LIQUIDATOR_ACCOUNT_OWNER=0x2c7536E3605D9C16a7a3D7b1898e529396a65c23
-LIQUIDATOR_ACCOUNT_OWNER_PRIVATE_KEY=0x4c0883a69102937d6231471b5dbb6204fe5129617082792ae468d01a3f362318
-EXPIRED_ACCOUNT_LIQUIDATION_DELAY_SECONDS=3600
-ENABLE_EXPIRATIONS=false
+# Solo Liquidations
+SOLO_LIQUIDATIONS_ENABLED=true
+SOLO_EXPIRATIONS_ENABLED=false
+SOLO_ACCOUNT_NUMBER=0
+SOLO_COLLATERAL_PREFERENCES=0,3,2
+SOLO_OWED_PREFERENCES=0,3,2
+SOLO_EXPIRED_ACCOUNT_DELAY_SECONDS=3600
+SOLO_MIN_ACCOUNT_COLLATERALIZATION=0.5
+SOLO_MIN_OVERHEAD_VALUE=100e36
+SOLO_LIQUIDATE_POLL_INTERVAL_MS=5000
 
-LIQUIDATOR_ACCOUNT_NUMBER=0
+# Perpetual Liquidations
+PERP_LIQUIDATIONS_ENABLED=false
+PERP_MIN_ACCOUNT_COLLATERALIZATION=0.5
+PERP_LIQUIDATE_POLL_INTERVAL_MS=5000
 
+# Global Settings
+WALLET_ADDRESS=0x2c7536E3605D9C16a7a3D7b1898e529396a65c23
+WALLET_PRIVATE_KEY=0x4c0883a69102937d6231471b5dbb6204fe5129617082792ae468d01a3f362318
```

Comment:
I don't think we should set this in .env

The process should crash if the user doesn't set a key